sidecar-injector/README.md

```diff
@@ -126,61 +242,272 @@ For this section, you'll work from a test namespace (test-namespace). Later you
+
+1. Set up environment variables modify to suite your needs e.g. use the same values from [kubernetes-conjur-demo](https://github.com/conjurdemos/kubernetes-conjur-demo)
+    ```bash
+    # required values
+    export TEST_APP_SERVICE_ACCOUNT=test-app-secretless
+    export containerMode=sidecar
+    export CONJUR_VERSION=4
+    export CONJUR_NAMESPACE_NAME=conjur-ktanekha
+    export CONJUR_ACCOUNT=my-account
+    export AUTHENTICATOR_ID=sidecar-test
+    export TEST_APP_NAMESPACE_NAME=sidecar-example-app
+
+
+    # derived values
+    CONJUR_APPLIANCE_URL="https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api"
+    CONJUR_AUTHN_URL="https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api/authn-k8s/${AUTHENTICATOR_ID}"
+    if [ ${CONJUR_VERSION} = '4' ]; then
+      CONJUR_AUTHN_LOGIN=${TEST_APP_NAMESPACE_NAME}/service_account/${TEST_APP_SERVICE_ACCOUNT}
+    else
+      CONJUR_AUTHN_LOGIN=host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${TEST_APP_NAMESPACE_NAME}/service_account/${TEST_APP_SERVICE_ACCOUNT}
+    fi
+    CONJUR_SSL_CERTIFICATE=$(follower_pod_name=$(kubectl -n ${CONJUR_NAMESPACE_NAME} get pods -l role=follower --no-headers | awk '{ print $1 }' | head -1); kubectl exec -n ${CONJUR_NAMESPACE_NAME} $follower_pod_name -- cat /opt/conjur/etc/ssl/conjur.pem)
+    ```
+
+1. Create test namespace
+    ```bash
+    ~$ kubectl create namespace ${TEST_APP_NAMESPACE_NAME}
+    ```
+
+1. Label the default namespace with `cyberark-sidecar-injector=enabled`
+    ```bash
+    ~$ kubectl label namespace ${TEST_APP_NAMESPACE_NAME} cyberark-sidecar-injector=enabled
+    ~$ kubectl get namespace -L cyberark-sidecar-injector
+    ```
+    ```
+    NAME                            STATUS    AGE       CYBERARK-SIDECAR-INJECTOR
+    default                         Active    18h
+    kube-public                     Active    18h
+    kube-system                     Active    18h
+    cyberark-sidecar-injector       Active    18h
+    sidecar-example-app             Active    18h       enabled
+    ```
+
+1. Create service account
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} create serviceaccount ${TEST_APP_SERVICE_ACCOUNT}
+    ```
+
+1. Create Conjur ConfigMap
+    ```bash
+    ~$ cat << EOL | kubectl -n ${TEST_APP_NAMESPACE_NAME} apply -f -
+    apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: conjur
+    data:
+      CONJUR_ACCOUNT: "${CONJUR_ACCOUNT}"
+      CONJUR_VERSION: "${CONJUR_VERSION}"
+      CONJUR_APPLIANCE_URL: "${CONJUR_APPLIANCE_URL}"
+      CONJUR_AUTHN_URL: "${CONJUR_AUTHN_URL}"
+      CONJUR_SSL_CERTIFICATE: |
+    $(echo "${CONJUR_SSL_CERTIFICATE}" | awk '{ print "    " $0 }')
+      CONJUR_AUTHN_LOGIN: "${CONJUR_AUTHN_LOGIN}"
     EOL
     ```
+
+### Deploy Authenticator Sidecar
+
+1. Deploy an app with the Authenticator Sidecar, take `test-app` app as an example
+    ```bash
+    ~$ kubectl -n ${TEST_APP_NAMESPACE_NAME} delete pod test-app
```

Comment:
You don't need this line if you're just running through this for the first time, especially unlikely since a previous command has you create the namespace.